packages/utils/src/__tests__/object.test.js

```diff
@@ -208,26 +201,26 @@ describe('object', () => {
     });
 
     describe('key mappers', () => {
-      const testData = [
-        ['string', [object1, object2], 'id', 'value', ['id1', 'id2']],
-        ['function', [object1, object2], object => object.value / 100, 'id', ['0.1', '0.2']],
-      ];
+      it('string', () => {
+        const result = reduceToDictionary([object1, object2], 'id', 'value');
+        expect(Object.keys(result)).toStrictEqual(['id1', 'id2']);
+      });
```

Comment:
And here is an example where I felt that parameterized tests actually made the test cases harder to read, so I reverted back the changes